frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -93,24 +102,6 @@ export const ProjectFeatureToggles = ({
 
     const featureLifecycleEnabled = useUiFlag('featureLifecycle');
 
-    const onTagClick = (tag: string) => {
```

Comment:
this PR makes this fn generic to work with tags/feature flag types or any other field